python/python_fastapi_server/routers/edr.py

```diff
@@ -123,12 +157,29 @@ def init_edr_collections(adaguc_dataset_dir: str = os.environ["ADAGUC_DATASET_DI
                             "name" in edr_parameter.attrib
                             and "unit" in edr_parameter.attrib
                         ):
-                            edr_params.append(
-                                {
-                                    "name": edr_parameter.attrib.get("name"),
+                            name = edr_parameter.attrib.get("name")
+                            layer = find_layer_configuration(
+                                adaguc_dataset_layers, name
+                            )
+                            # If the layer is present in the adaguc dataset configuration, continue.
+                            if layer is not None:
```

Comment:
Do we need to throw and error of no layer is matched, as the EDR line is now silently ignored?

Reply:
A warning is printed instead.